three/index.d.ts

```diff
@@ -1996,7 +1998,7 @@ declare namespace THREE {
         mimeType: MimeType;
         path: string;
         responseType: string;
-        withCredentials: string        
+        withCredentials: string
```

Comment:
Can you add a `tslint.json` file to the directory? This would catch the missing semi-colon here.